File: backend/Core/DTOs/HomeInsightDto.cs
using Core.Enums;

namespace Core.DTOs
{
    public sealed class HomeInsightDto
    {
        public string Key { get; init; } = null!;
        public string Title { get; init; } = null!;
        public string Subtitle { get; init; } = null!;
        public InsightIconEnum Icon { get; init; }
        public InsightScopeEnum Scope { get; init; }
    }
}

File: mobile/src/app/models/homeinsight.dto.ts
import { InsightScopeEnum } from '../enums/insightscope.enum';
import { HomeInsightIconEnum } from '../enums/homeinsighticon.enum';

export interface HomeInsightDto {
  key: string;
  title: string;
  subtitle: string;
  icon: HomeInsightIconEnum;
  scope: InsightScopeEnum;
}
